pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -209,6 +240,10 @@ public String toString() {
     return super.toString();
   }
 
+  public Configuration getPinotFSFactoryClasses() {
```

Comment:
Is this actually used?

Reply:
I see that we usually have methods both for get and set for each configuration we wire in, so I added that for consistency. It isn't currently used.